sdk/js/src/token_bridge/redeem.ts

```diff
@@ -219,3 +235,121 @@ export async function redeemOnAlgorand(
     senderAddr
   );
 }
+
+/**
+ * This basically just submits the VAA to Near
+ * @param client
+ * @param tokenBridge Token bridge ID
+ * @param vaa The VAA to be redeemed
+ * @returns Transaction ID(s)
+ */
+export async function redeemOnNear(
+  client: nearAccount,
+  tokenBridge: string,
+  vaa: Uint8Array
+): Promise<String> {
```

Comment:
I think this should return some kind of receipt, but maybe that's what this is?

Reply:
it returns the account that represents the fungible token